src/v/cluster/rm_stm.cc

```diff
@@ -487,6 +487,18 @@ ss::future<tx_errc> rm_stm::do_prepare_tx(
         co_return tx_errc::fenced;
     }
 
+    if (_log_state.tx_seqs.contains(pid)) {
```

Comment:
> cluster: Prepare prepared for transaction_ga
We are donna do not use prepare after ga

Can you fix this sentence ^ in the commit message

Reply:
update the commit message